src/pointers.js

```diff
@@ -7,5 +7,6 @@ export default function(events, node) {
     if (node === undefined) node = events.currentTarget;
     events = events.touches || [events];
   }
+console.warn({events, node})
```

Comment:
Did you mean to commit this console.warn?

Reply:
oops! thanks for catching it.